Pascal/Pascal.sublime-syntax

```diff
@@ -47,7 +47,7 @@ contexts:
             0: punctuation.definition.comment.pascal
           push:
```

Comment:
same here. Not sure how necessary the whole `(?!\G)` is, maybe it can be removed?